index.html

```diff
@@ -2570,39 +2589,27 @@
             <tr>
               <td><dfn><code id="idl-def-RTCDtlsRole.auto">auto</code></dfn></td>
               <td>
-                <p>The DTLS role is determined based on the resolved ICE role: the
-                <code>controlled</code> role acts as the DTLS client, the
-                <code>controlling</code> role acts as the DTLS server. Since
-                <code><a>RTCDtlsRole</a></code> is initialized to <code>auto</code> on
-                construction of an <code><a>RTCDtlsTransport</a></code> object,
-                <code>transport.getLocalParameters().RTCDtlsRole</code> will have an
-                initial value of <code>auto</code>.</p>
+                <p>The DLTS role is determined based on the resolved ICE role:
+                the ICE <code>controlled</code> role acts as the DTLS client and
+                the ICE <code>controlling</code> role acts as the DTLS server.
+                <code>getLocalParameters().role</code> always has a
+                value of <code>auto</code>.</p>
```

Comment:
I think that's confusing. Don't get me wrong, I understand the concept and I do realise this is intended for interop but the reader may ask why it's included in the first place if it's always *auto* anyway.

Furthermore, why is the role included in the parameters and not separate and static like the SCTP capabilities are? In the SCTP transport, *parameters* are separated into the capabilities (static) and the port (not static). However, for API consistency, why is that not the case for the DTLS transport's role as well?